tests/test_reader.py

```diff
@@ -70,3 +70,40 @@ def test_mapping_by():
         }
     }
     assert expected == mapping
+
+
+def test_get_rdm_subjects(running_app, create_subject_data):
+    subjects = [
+        create_subject_data(
+            input_={
+                "id": "http://example.org/foo/0",
+                "scheme": "foo",
+                "subject": "0",
+            }
```

Comment:
Could have just called `create_subject_data()` for this one, no? Just double-checking myself. The consistency here looks better anyway.

Reply:
Yes that's right we could have called it without any arguments and the default values would have kicked in and amounted to the same thing. Indeed, passing the input explicitly was for consistency and make it visible from the test where the "0" in the assertion below came from. 

I am debating the usefulness of these defaults in the end, since I mostly don't use them. We'll see if I get rid of them altogether.